cypress/e2e/Courses/Certificates/StandardPlanUnawardedCourseCertificate/StandardPlanUnawardedCourseCertificate.js

```diff
@@ -0,0 +1,102 @@
+import { Given, When, Then } from '@badeball/cypress-cucumber-preprocessor';
+
+Given('a standard plan user has some completed unawarded premium courses', () => {
+  cy.fixture('/Certificate/CourseCertificate/credentials').then((credentials) => {
+    cy.log(credentials);
+    cy.loginAccount(credentials.standard_unawarded);
+  });
+});
+
+// START Scenario: Standard plan users should see the call to action "Review course" for completed and unawarded premium courses
+Given('the premium courses page has been navigated to', () => {
+  cy.visit('/premium-courses');
+});
+
+Given('an iOS mobile device is not being used', () => {
+  this.userAgent = undefined;
+});
+
+When('the user selects the "Completed" tab', () => {
+  cy.intercept('GET', '/api/feed/premium-courses?enrollmentStatus=completed').as('completedCourses');
+  cy.contains('Completed').click();
+});
+
+Then('the user should see for unawarded courses cards the warning style', () => {
+  cy.intercept('GET', '/api/feed/premium-courses?enrollmentStatus=completed').as('completedCourses');
+  cy.wait('@completedCourses').then((interception) => {
+    interception.response.body.data.forEach((course, index) => {
+      if (!course.isAwarded) {
+        cy.getByTestId('status-completed').eq(index)
+        .contains('1/1')
+        .should('have.css', 'color', 'rgb(234, 120, 14)');
```

Comment:
Can we use the Hexa version? it could be easier to replace it in the future if we do a full search of this color.

Reply:
Ignore this one, the test will fail if we change the color

Reply:
tests do not allow hex strings

Reply:
wow!